cra-memory-considerate-loading/src/utils/hooks.js

```diff
@@ -14,25 +14,25 @@
  * limitations under the License.
  */
 
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 
-const unsupportMessage = 'The Memory Status API is not supported on this platform.';
+export const CLIENT_SIDE_UNSUPPORT_MESSAGE = 'The Memory Status API is not supported on this platform.';
+const SERVER_SIDE_UNSUPPORT_MESSAGE = 'This Memory React Hook uses Window interface so it\'s not possible to use this on Server Side Rendering';
 
 // Tune these for your application
-const MAX_MEMORY_LIMIT = 50 * 1048576; // 50MB
-const MAX_PERCENT_THRESHOLD = 90;
+export const MAX_MEMORY_LIMIT = 50 * 1048576; // 50MB
+export const MAX_PERCENT_THRESHOLD = 90;
+
```

Comment:
I think we should avoid inline exporting values here (`export const <foo>`) and instead export these all as part of the final `export { useMemoryStatus };` line at the end. It gives us a central place to see what's actually being exported.

Reply:
Yes for sure. Sounds like a good idea.
Let me check other hooks as well and update them accordingly.